src/components/Facility/FacilityHome.tsx

```
@@ -1,39 +1,18 @@
 import careConfig from "@careConfig";
-import {
-  Tooltip,
-  TooltipContent,
-  TooltipProvider,
-  TooltipTrigger,
-} from "@radix-ui/react-tooltip";
 import { useQuery, useQueryClient } from "@tanstack/react-query";
 import { useMutation } from "@tanstack/react-query";
+import { Hospital } from "lucide-react";
 import { Edit2, Hospital, MapPin, MoreVertical, Settings } from "lucide-react";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Hospital'. Consider to delete it or rename it.

'Hospital' is defined here:


(lint/suspicious/noRedeclare)